lib/ts/recipe/thirdpartypasswordless/index.ts

```diff
@@ -70,6 +70,9 @@ export default class Wrapper {
     static EmailVerification = (prop?: any) =>
         ThirdPartyPasswordless.getInstanceOrThrow().getFeatureComponent("emailverification", prop);
     static EmailVerificationTheme = EmailVerificationTheme;
+
+    static LinkClicked = (prop?: any) =>
+        ThirdPartyPasswordless.getInstanceOrThrow().getFeatureComponent("linkClickedScreen", prop);
 }
 
```

Comment:
We should also expose ThirdPartyCallback screen in this, ThirdParty and ThirdPartyEmailPassword recipe?